internal/querycoordv2/job/job_release.go

```diff
@@ -89,14 +89,14 @@ func (job *ReleaseCollectionJob) Execute() error {
 		return errors.Wrap(err, msg)
 	}
 
+	job.targetObserver.ReleaseCollection(req.GetCollectionID())
+	waitCollectionReleased(job.dist, job.checkerController, req.GetCollectionID())
+	tryReleaseCollection(job.ctx, job.meta, job.cluster, req.GetCollectionID())
```

Comment:
`tryReleaseCollection` is confused, prefer to `cleanCollectionMetaCache`

Reply:
done